aea/configurations/schemas/skill-config_schema.json

```diff
@@ -106,37 +106,16 @@
     }
   },
   "definitions": {
-    "handlers": {
-      "type": "object",
-      "additionalProperties": false,
-      "uniqueItems": true,
-      "patternProperties": {
-        "^[^\\d\\W]\\w*\\Z": {
-          "$ref": "#/definitions/handler"
-        }
-      }
-    },
-    "behaviours": {
+    "skill_component_list": {
       "type": "object",
-      "uniqueItems": true,
       "patternProperties": {
         "^[^\\d\\W]\\w*\\Z": {
-          "$ref": "#/definitions/behaviour"
+          "$ref": "#/definitions/skill_component_configuration"
         }
       }
     },
-    "models": {
+    "skill_component_configuration": {
       "type": "object",
```

Comment:
What about `"additionalProperties": false,`?

Reply:
we use `patternProperties` so that constraint does not apply.